examples/tutorial/x/blog/handlers_test.go

```diff
@@ -966,3 +1006,76 @@ func TestSetProfileMsgHandlerDeliver(t *testing.T) {
 		},
 	)
 }
+
+func TestQuery(t *testing.T) {
```

Comment:
This looks good to me.
You can rename to TestBlogQuery.

I would add one query with no results as well.
We can find a blog by slug, or find no blog by slug.

It would also be good to add ability to find a blog by author....
This involves adding a secondary index. I think it's only set up for on value per index, so at just the first (main) author is fine for the tutorial.

That would be a good example to show.